style.css

```diff
@@ -1,3 +1,164 @@
+/* Generic Styling */ 
+
 * {
 	box-sizing: border-box;
-}
\ No newline at end of file
+	font-family: Helvetica;
+}
+
+h1 {
+	font-family: cursive;
+	font-size: 50px;
+	margin: 20px auto;
+}
+
+h2 {
+	font-size: 30px ;
+}
+
+/*This is the header section*/
+
+header {
+	display: flex;
+	flex-direction: column;
+	background-color: white;	
+}
+
+
+nav {
+	display: flex;
+		margin: 20px auto;
+		gap: 20px;
+		justify-content: space-evenly;
+}
+
+nav a {
+	text-decoration: none;
+	font-weight: bold;
+	color: black;
+}
+
+/* Introduction/Welcome  section*/
+
+.introduction_section {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-end;
+	justify-content: center;
+	column-gap: 20px;
+	width: 100%;
+	height: 500px;
+	background-image: url(images/horse_head_nebula.jpg); /* a place-holder image */ 
+	background-size:cover; /* allows to get the background in the whole section*/
+}
+
+.introduction_section h2 {
+	background-color: white;
+	color: black;
+	font-variant: small-caps;
+	margin-right: 50px;
+}
+
+.introduction_section p {
+	background-color: black;
+	color: white;
+	margin-right: 50px;
+	font-size: 25px;
+}
+
+
+/* About me section */
+
+.aboutMe_section {
+	display: flex;
+  flex-direction: column;
+/*	align-items: center;*/
+}
+
+.aboutMe_card {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: flex-start;
+	margin-left: 50px;
+	gap: 20px;
+}
+
+.aboutMe_section img {
+	width: 200px;
+}
+
+
+.aboutMe_section h2 {
+		margin-left: 50px;
+	}
+
+.aboutMe_text {
+	display: flex;
+	flex-direction: column;
+	align-items: center;		
+	max-width: 500px;
+	
+}
+
+/* Portfolio section  */ 
+
+.portfolio_section h2 {
+	margin-left: 50px;
+}
+
+.project_cards_container {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: space-between;
```

Comment:
this is leaving an unusual result on row two. Perhaps a result of the busyness of the page - could you explore other options here?